### ISSUE TITLE
Error...  sh: --injectSet: not found

### ISSUE THREAD

**Issue by Mikewins1** [sud0nick/PortalAuth#20]
Every time I try to clone a site I get error that says this   sh: --injectSet: not found

wifipineapple nano

**Comment by jairwen** [comment]
I guess you used sd card. In includes/scripts/PortalCloner,  self.basePath = '/pineapple/modules/PortalAuth/' => /sd/modules/... 

> Every time I try to clone a site I get error that says this sh: --injectSet: not found
> 
> wifipineapple nano

**Comment by sud0nick** [comment]
> I guess you used sd card. In includes/scripts/PortalCloner, self.basePath = '/pineapple/modules/PortalAuth/' => /sd/modules/...

No need to change that.  The way the Pineapple works is it creates a symlink to the module on the SD card in /pineapple/modules/.  Using that directory is guaranteed to work whether an SD card is present or not.

**Comment by sud0nick** [comment]
> Every time I try to clone a site I get error that says this sh: --injectSet: not found
> 
> wifipineapple nano

Look at the portalclone.py script in /pineapple/modules/PortalAuth/scripts and see if there are any encoding issues.  I'm not able to reproduce this problem but while attempting to I noticed one of my scripts didn't unzip properly and left a strange character in the middle of the file.  Once I removed it the script ran.  I'll keep looking into it, though.
